packages/docs/README.md

```diff
@@ -0,0 +1,25 @@
+# EmbeddedChat Documentation
+
+This is the official documentation website of EmbeddedChat
+
+### Installation
+
+```
+$ yarn install
+```
+
+### Local Development
+
+```
+$ yarn run start
```

Comment:
Modify package.json and use the script `dev`, so it can be opened with `yarn dev` to have uniformity across the project

Reply:
done